octavia_f5/restclient/as3objects/service.py

```diff
@@ -130,31 +148,63 @@ def get_service(listener, cert_manager):
             default_pool = m_pool.get_name(listener.default_pool_id)
             service_args['pool'] = default_pool
 
+            """ only consider Proxy pool, everything else is determined
+                by listener type """
```

Comment:
Why docstring instead of a comment?

Reply:
multiline

Reply:
You can just write a comment and then use the 'Fill paragraph' functionality of PyCharm ;)